rules/change-message-size-restrictions-exchange-online/rule.md

```diff
@@ -32,7 +32,7 @@ Of course, this can also be done with the Exchange Management Shell, for example
 
 <!--StartFragment-->
 
-```
+```console
```

Comment:
Hey @tombui99 

I don't think this is a supported language: https://prismjs.com/